routes/paymentRouter.js

```diff
@@ -0,0 +1,16 @@
+// Set your secret key: remember to change this to your live secret key in production
+// See your keys here: https://dashboard.stripe.com/account/apikeys
+const router = require('express').Router()
+
+const stripe = require('stripe')(process.env.STRIPE_SECRET);
+
+router.post('/charge', async (req,res) => {
+    let transaction = req.body
+    const paymentIntent = await stripe.paymentIntents.create({
+        amount: transaction.amount,
+        currency: transaction.currency || 'usd',
```

Comment:
Is it a good idea to default to USD if the currency isn't specified? I'm concerned that if I want to buy a pair of boxers for N500 I might get charged USD500 if the store isn't set up right!

Reply:
that's true. Will fix